### DISCUSSION TITLE
Hide & Show Components | 2 conditions

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#5074] [Help] [answered]
Hi there,

I can't seem to get my conditions running like I want them to.
I created 3 departments for my "departments" attribute I have on every user.
My first condition is:
`Hide component IF {{ Current User.Dept }} Not equals Binding Finance`
The condition works perfectly.
Second condition won't work no matter what.
Is there a limit on conditions per component?

I'm trying to let the ADMIN Role see every table of the 3 departments.

**Comment by mjashanks** [comment, marked as answer]
Hi @kasgithub8. I just tried this, and it seems to work ok for me:

I'#m using colours so we can see it properly. The second condition takes precedence.

![image](https://user-images.githubusercontent.com/3524181/160381159-7e951877-cf8d-4983-aafc-d101d7f0703b.png)

**Reply by mjashanks** [reply]
Also, I tried this with show/hide component. Initially I thought i'd recreated your issue.

But then I realised that it was only happening inside the builder preview, because the component was still selected..

**Still Selected**
![image](https://user-images.githubusercontent.com/3524181/160381789-11e10671-0734-4408-b5e4-1c1c11fadec7.png)

**Not Selected**
![image](https://user-images.githubusercontent.com/3524181/160381833-fbd9577c-d88b-4241-aa24-bbff1c3d88b8.png)

We do this on purpose so that the component always displays when selected
